wp1/web/sites.py

```diff
@@ -0,0 +1,27 @@
+import flask
+import mwclient
+from datetime import timedelta
+from wp1.web.redis import get_redis
+
+sites = flask.Blueprint('sites', __name__)
+_ua = 'WP1.0Bot/3.0. Run by User:Audiodude. Using mwclient/0.9.1'
+
+
+@sites.route('/')
+def get_sites():
+  redis = get_redis()
+  if redis.get('sites'):
+    return {'data': redis.get('sites').decode('utf-8').split(',')}
+
+  site = mwclient.Site('meta.wikimedia.org', clients_useragent=_ua)
+  result = site.api('sitematrix')
+  response = []
```

Comment:
```suggestion
  sites = []
```

It's good to use the same variable name here as in the caching code above, so that readers can understand we're referring to the same data.